superset-frontend/src/dashboard/components/SaveModal.tsx

```diff
@@ -185,7 +207,7 @@ class SaveModal extends React.PureComponent {
             <div className="m-l-25 m-t-5">
               <Checkbox
                 checked={this.state.duplicateSlices}
-                onChange={this.toggleDuplicateSlices}
+                onChange={() => this.toggleDuplicateSlices()}
```

Comment:
still a bit confused why we needed to change this since we bind this in the constructor for this function

Reply:
We get an warning of that argumeng "boolean | undefined" where we do not use it. That's why I added callback here